src/dotty/tools/dotc/transform/LazyVals.scala

```diff
@@ -367,7 +367,7 @@ class LazyVals extends MiniPhaseTransform with IdentityDenotTransformer {
             appendOffsetDefs += (companion.moduleClass -> new OffsetInfo(List(offsetTree), ord))
         }
 
-        val containerName = ctx.freshName(x.name ++ StdNames.nme.LAZY_LOCAL).toTermName
```

Comment:
@DarkDimius This is just DRY in order not to do the same low-level name plumbing over and over again.